### PR TITLE
Fix travis tests, for realsies

### PR DESCRIPTION
**Pull request by Meroje**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,12 +28,11 @@
     "require-dir": "^0.3.2",
     "run-sequence": "^1.1.5",
     "uglify-save-license": "^0.4.1",
-    "webdriver-manager": "^12.0.6",
+    "webdriver-manager": "^10.3.0",
```
Comment (fdubost) [inline package.json]: downgrade ? 👀

File: test/protractor.conf.js [hunk 2]
```diff
@@ -1,12 +1,22 @@
 'use strict';
 
+var path = require('path');
+var SeleniumConfig = require('webdriver-manager/built/lib/config').Config;
+var SeleniumChrome = require('webdriver-manager/built/lib/binaries/chrome_driver').ChromeDriver;
+var seleniumChrome = new SeleniumChrome();
+seleniumChrome.versionCustom = '2.30';
```
Comment (fdubost) [inline test/protractor.conf.js]: Why we have to use a custom version❓

File: gulp/e2e-tests.js [hunk 3]
```diff
@@ -24,8 +31,13 @@ gulp.task('protractor-only', ['webdriver-update'], function (done) {
     'test/e2e/**/*.js'
   ];
 
+  var seleniumStandAlone = new SeleniumStandAlone();
+  seleniumStandAlone.versionCustom = '3.4.0';
```
Comment (fdubost) [inline gulp/e2e-tests.js]: Why we have to use a custom version❓

File: package.json [hunk 4]
```diff
@@ -28,12 +28,11 @@
...
     "wiredep": "^2.2.0"
   },
   "devDependencies": {
-    "httpbackend": "^2.0.0",
-    "protractor": "^3.3.0"
```
Comment (fdubost) [inline package.json]: Why is it removed ? 🙄

Comment (Meroje) [inline package.json]: protractor is not a direct dependency of the project

File: test/protractor.conf.js (hunk 2 above)
Comment (Meroje) [inline test/protractor.conf.js]: 2.30 is not yet the default on protractor which still uses 2.26

File: gulp/e2e-tests.js (hunk 3 above)
Comment (Meroje) [inline gulp/e2e-tests.js]: not yet the default on protractor, we need 3.4 to support chrome headless

File: package.json (hunk 1 above)
Comment (Meroje) [inline package.json]: This is the same requirement as protractor, to be sure npm deduplication works and the path to selenium jar is the same between the download and the usage from protractor

File: package.json (hunk 4 above)
Comment (fdubost) [inline package.json]: Of which package is it the dependency ?

File: package.json (hunk 1 above)
Comment (fdubost) [inline package.json]: Ok 😷

**Comment by Meroje** [comment]
This is a bit hacky while protractor doesn't have out of the box support for chrome headless, but tests run and work can resume on #28

File: package.json (hunk 4 above)
Comment (Meroje) [inline package.json]: [gulp-protractor](https://github.com/mllrsohn/gulp-protractor/blob/9254186550e665a14491b5b53d4a9670d4fa4c24/package.json#L41) required [here](https://github.com/Meroje/GithubTeamReviewer/blob/ce2664c711fa3fb7a9278ee889c083ba9d21499d/package.json#L18)

Comment (fdubost) [inline package.json]: Ok, good 👍

**Comment by Meroje** [comment]
@fdubost can we merge this before #30 ?

**Pull request merged**